fxa-auth-db-server/docs/API.md

```diff
@@ -1761,3 +1766,128 @@ Content-Length: 2
     * Body : {"code":"InternalError","message":"...<message related to the error>..."}
 ```
 
+## createSigninCode : `PUT /signinCodes/:code`
+
+Create a user-specific, time-limited, single-use code
+that can be used for expedited sign-in.
+
+### Example
+
+```
+curl \
+    -v \
+    -X PUT \
+    -H "Content-Type: application/json" \
+    -d '{"uid":"fdea27c8188b3d980a28917bc1399e47","createdAt":1494253830983}' \
+    http://localhost:8000/signinCodes/1234567890ab
+```
+
+### Request
+
+* Method : `PUT`
+* Path : `/signinCodes/<code>
+    * `code` : hex48
+* Params:
+    * `uid` : hex128
+	* `createdAt` : epoch
+
+### Response
+
+```
+HTTP/1.1 200 OK
+Content-Type: application/json
+Content-Length: 2
+
+{}
+```
+
+* Status Code : `200 OK`
+    * Content-Type : `application/json`
+    * Body : `{}`
+* Status Code : `409 Conflict`
+    * Conditions: if the specified sign-in code already exists
+    * Content-Type : `application/json`
+    * Body : `{"errno":101,"message":"Record already exists"}`
+* Status Code : `500 Internal Server Error`
+    * Conditions: if something goes wrong on the server
+    * Content-Type : `application/json`
+    * Body : `{"code":"InternalError","message":"..."}`
+
+## useSigninCode : `DELETE /signinCodes/:code`
+
+Use a sign-in code.
+
+### Example
+
+```
+curl \
+    -v \
+    -X DELETE \
+    http://localhost:8000/signinCodes/1234567890ab
```

Comment:
Same question I asked @vbudhram, what should I change it to? Fwiw, a `GET` request that deletes items from the database seems super weird to me. :smile: